### ISSUE TITLE
Used letters is hard to see

### ISSUE THREAD

**Issue by eson57** [monich/harbour-wordle#8]
Could you possible change the colour of used letters, or maybe darken them even more, because they are quite difficult to distinguish from the unused

![Skärmklipp_20220203_001](https://user-images.githubusercontent.com/2996154/152481135-19aba8c0-de57-49fd-98a2-045059034271.png)
?

**Comment by monich** [comment]
I have to agree that those are hard to distinguish

**Comment by monich** [comment]
Does 1.0.5 look better?

**Comment by eson57** [comment]
Much better. Thanks!

**Comment by eson57** [comment]
I think you forgot to change the enter key.  ;)
The back key also don't look right, on a clean game with no words entered.

![Skärmklipp_20220217_001](https://user-images.githubusercontent.com/2996154/154416315-921951f1-6ad5-42a3-80cc-641b8001870e.png)

**Comment by monich** [comment]
> I think you forgot to change the enter key. ;)
> The back key also don't look right, on a clean game with no words entered.

Those get enabled/disabled based on the state of the input. On your screenshot (partial input), backspace is enabled and enter is disabled. When the input row is empty, both are disabled. When it's full, both are enabled. That was the idea.

**Comment by eson57** [comment]
Yes, I understand that, but I think it looks kind of weird with a third grey nuance. Shouldn't the disabled grey be the same as the used grey?
